sumologic/resource_sumologic_cse_automation_test.go

```diff
@@ -0,0 +1,126 @@
+package sumologic
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-sdk/helper/resource"
+	"github.com/hashicorp/terraform-plugin-sdk/terraform"
+)
+
+func TestAccSumologicSCEAutomation_create_update(t *testing.T) {
+	SkipCseTest(t)
+
+	var Automation CSEAutomation
+	nPlaybookId := "63ece953d5f0cb2ec4d5794e"
+	nCseResourceType := "INSIGHT"
+	nExecutionTypes := []string{"NEW_INSIGHT"}
+	nEnabled := true
+	uExecutionTypes := []string{"ON_DEMAND"}
+	uEnabled := false
+	resourceName := "sumologic_cse_automation.automation"
+	resource.Test(t, resource.TestCase{
+		PreCheck:     func() { testAccPreCheck(t) },
+		Providers:    testAccProviders,
+		CheckDestroy: testAccCSEAutomationDestroy,
+		Steps: []resource.TestStep{
```

Comment:
Should there be an update test in here as well? I see the two create ones.

Reply:
the test also does an update, see step:
```
			{
				Config: testCreateCSEAutomationConfig(nPlaybookId, nCseResourceType, uExecutionTypes, uEnabled),
				Check: resource.ComposeTestCheckFunc(
					testCheckCSEAutomationExists(resourceName, &Automation),
					testCheckAutomationValues(&Automation, nPlaybookId, nCseResourceType, uExecutionTypes, uEnabled),
				),
			},
```
it is updating the execution type and enabled field

Reply:
Oh, I see. I didn't read quite right to notice the difference between 'nExecutionTypes' and 'uExecutionTypes'.